pkg/db/db.go

```diff
@@ -65,6 +66,19 @@ func Init(cacheDir string) (err error) {
 		return xerrors.Errorf("failed to mkdir: %w", err)
 	}
 
+	// bbolt sometimes occurs the fatal error of "unexpected fault address".
+	// In that case, the local DB should be broken and needs to be removed.
+	debug.SetPanicOnFault(true)
```

Comment:
I'm thinking whether we should set this flag in the upstream trivy project. Somewhere in the main() method.

Reply:
It is reasonable to catch faults at an unexpected (non-nil) address from BoltDB because they use a memory-mapped file, but I'm not sure we should catch all faults in trivy. So, I set this flag only in this function. Then, this flag is disabled in the `defer`. 
https://github.com/aquasecurity/trivy-db/pull/35/files#diff-ac4e04b0d1bbf8235592bdbb82938144R79

Does it make sense to you? Actually, I'm not sure what we should do here.

Reply:
Yup. Got your point. If this code is not accessed by multiple goroutines we could leave it as it is.

Reply:
Thanks. This is a function to initialize a database, so it is intended to be called once at the beginning.